### PR TITLE
Add services finder and email signup

### PR DESCRIPTION
**Pull request by bilbof**
https://trello.com/c/e53rC9Ee/287-publish-services-finder-from-rummager

This can be published with the following rake task:

```
publishing_api:publish_finder FINDER_CONFIG=services.yml EMAIL_SIGNUP_CONFIG=services_email_signup.yml
```

### CODE REVIEW

File: config/finders/services.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+analytics_identifier:
+base_path: "/services"
+content_id: f6d779ac-5f78-413d-a1ff-da391944e6ec
+content_purpose_document_supertype: navigation
```
Comment (tijmenb) [inline config/finders/services.yml]: I think this supertype and the others aren't used. Can we remove them if that's the case? Might be worth looking at the other entries as well.

Comment (bilbof) [reply config/finders/services.yml]: Thanks, @tijmenb! You're right - I removed them.

File: config/finders/services.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  sort:
+  - key: title
+    name: A-Z
+  - default: true
+    key: "-popularity"
+    name: Most viewed
+  - key: "-relevance"
+    name: Relevance
+document_type: finder
+first_published_at: '2019-01-15T13:00:00.000+00:00'
```
Comment (sihugh) [inline config/finders/services.yml]: we can remove this

File: config/finders/services.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+links: {}
+locale: en
+public_updated_at: '2019-01-15T13:00:00.000+00:00'
```
Comment (sihugh) [inline config/finders/services.yml]: we can remove this

File: config/finders/services.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+publishing_app: finder-frontend
```
Comment (sihugh) [inline config/finders/services.yml]: this should be `rummager`

File: config/finders/services.yml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+rendering_app: finder-frontend
+schema_name: finder
+title: Services
+updated_at: '2019-01-15T13:00:00.000+00:00'
```
Comment (sihugh) [inline config/finders/services.yml]: we can remove this

File: config/finders/services.yml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+  - key: title
+    name: A-Z
+  - default: true
+    key: "-popularity"
+    name: Most viewed
+  - key: "-relevance"
+    name: Relevance
+document_type: finder
+first_published_at: '2019-01-15T13:00:00.000+00:00'
+links: {}
```
Comment (sihugh) [inline config/finders/services.yml]: we can remove this

File: config/finders/services.yml [hunk 7]
```diff
@@ -0,0 +1,57 @@
+---
+analytics_identifier:
```
Comment (sihugh) [inline config/finders/services.yml]: we can remove this

**Review by sihugh — CHANGES_REQUESTED** [review]
There's just a few bits that publishing api will generate for us.  They'll be present in content store, but we don't need to provide them here.

**Comment by bilbof** [comment]
Thank you, @sihugh, I removed those unnecessary fields.

**Review by sihugh — APPROVED** [review]
🚀

**Pull request merged**